ios/Hubbub/LoginViewController.swift

```diff
@@ -9,6 +9,8 @@
 import Firebase
 import p2_OAuth2
 import UIKit
+import Crashlytics
```

Comment:
Do we actually want to commit the stuff in this file, or was it just for debugging/verifying the integration?